### DISCUSSION TITLE
pressing the Stop button does not disable blocking fuctionality

### DISCUSSION THREAD

**Discussion by sudo-joe** [celzero/rethink-app#1772] [General]
Hello,

imported a Proton-VPN.conf in RethinkDNS. All other settings are untouched(default settings after installation).
But this way, Rething says for example: „Unable to resolve host „auroraoss.com“: No address assiciated with hostname“

Therefore i pressed the button Stop (in RethinkDNS) in order to be able to check if internet connectivity does work...
But none of the apps installed in this GrapheneOS profile does work like intended.

Only after de-installing RethinkDNS completely, all apps can access the internet.

Any ideas what I am doing wrong?

Thank you!

**Comment by ignoramous** [comment]
> Rething says for example: „Unable to resolve host „auroraoss.com“: No address assiciated with hostname“

Sounds like DNS resolutions are failing. Where (in which UI screen) do you see this error, btw?

Are you using WireGuard in *Advanced* or *Simple* mode?

Can you switch to `System DNS` from *Configure* -> *DNS* and see if things work?
